hackathon_site/event/jinja2/event/landing.html

```diff
@@ -0,0 +1,306 @@
+                    <p class="scheduleTime">10AM</p>
+                    <p>Opening Ceremonies</p>
+                </div>
+                <div class="scheduleRow">
+                    <p class="scheduleTime">11AM</p>
+                    <p><strong>Hacking Begins</strong></p>
+                </div>
+                <div class="scheduleRow">
+                    <p class="scheduleTime">Various</p>
+                    <p>Workshops & Activities</p>
+                </div>
+            </div>
+
+            <div class="col s12 m5 l6">
+                <h4 class="center">Sunday</h4>
+
+                <div class="scheduleRow">
+                    <p class="scheduleTime">11AM</p>
+                    <p><strong>Hacking Ends!</strong></p>
+                </div>
+                <div class="scheduleRow">
+                    <p class="scheduleTime">12PM</p>
+                    <p>Judging</p>
+                </div>
+                <div class="scheduleRow">
+                    <p class="scheduleTime">3PM</p>
+                    <p>Closing Ceremonies</p>
+                </div>
+                <div class="scheduleRow">
+                    <p class="scheduleTime">4PM</p>
+                    <p>End</p>
+                </div>
+            </div>
+        </div>
+        <div class="m12 center">
+            <a class="btn-large waves-effect waves-light colorBtn" rel="noopener" target="_blank">Full schedule</a>
+        </div>
+    </div>
+</div>
+<br /><br />
+
+<div class="container">
+    <div class="section scrollspy" id="faq">
+        <div class="row">
+            <div class="col s12 m12 center">
+                <h3 class="center">FAQ</h3>
+            </div>
+            <div class="col s12 m6">
+                <ul class="collapsible">
+                    <li>
+                        <div class="collapsible-header">
+                            What is a hackathon?
+                        </div>
+                        <div class="collapsible-body">
+                            <span>A hackathon is a big bonanza where people from everyone doing everything come together to create projects together with the newest and most exciting tech.</span>
+                        </div>
+                    </li>
+                    <li>
+                        <div class="collapsible-header">
+                            How much does it cost?
+                        </div>
+                        <div class="collapsible-body">
+                            <span>$0! IEEE UofT is a non-profit organization that aims to provide students accessible technical events.</span>
+                        </div>
+                    </li>
+                    <li>
+                        <div class="collapsible-header">
+                            When can I expect to hear back after I applied?
+                        </div>
+                        <div class="collapsible-body">
+                            <span
```

Comment:
Tag got a little bit weird because of prettier. Should we fix it and avoid using prettier, or keep using prettier but have weird looking tags?